GVFS/GVFS.GVFlt/GVFS.GVFlt.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
-  <Import Project="$(SolutionDir)\GVFS\GVFS.Build\GVFS.cs.props" />
+  <Import Project="..\GVFS.Build\GVFS.cs.props" />

```

Comment:
wondering why this path is changed.

Reply:
Most likely because Visual Studio like to do things like this 😀 😢

Reply:
This is because SolutionDir is not defined when building the project, only when building the solution.

Reply:
One thing I've done in the past is to define an MSBuild property `<RepoRoot>` in a Directory.Build.props file in the root of the repository.

Projects can use `$(RepoRoot)` if they want to use an absolute path (rather than a maybe more brittle relative one), without needing the Solution build to set `$(SolutionDir)`.